gem/frank-cucumber.gemspec

```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |s|
 
   s.add_dependency( "cucumber", ["~>1.3.10"] )
   s.add_dependency( "rspec", ["~>2.14.1"] )
-  s.add_dependency( "sim_launcher", ["~>0.4.6"] )
```

Comment:
were these dependency updates necessary as part of the 64 bit work? If not I'd like them in a separate Pull Request.